sale_end_partner/tests/test_sale_end_partner.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleEndPartner(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.product = cls.env["product.product"].create(
+            {
+                "name": "Test Product",
+                "type": "service",
+            }
+        )
+        cls.partner = cls.env["res.partner"].create(
+            {
+                "name": "Test Customer",
+            }
+        )
```

Comment:
Please avoid taking more lines than necessary.
```suggestion
        cls.partner = cls.env["res.partner"].create({"name": "Test Customer"})
```

Reply:
Done!